Formula/edbg.rb

```diff
@@ -0,0 +1,22 @@
+class Edbg < Formula
+  desc "Utility for programming ARM-based MCUs though CMSIS-DAP SWD interface"
+  homepage "https://github.com/ataradov/edbg"
+  url "https://github.com/ataradov/edbg/archive/master.tar.gz"
```

Comment:
In Homebrew we only accept stable releases of software, and do not built from repo's master or alpha/beta versions.